climpred/bias_reduction.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+import xarray as xr
+
+from climpred.metrics import Metric
+
+
+def _mean_bias_reduction_quick(hind, bias, dim):
+    """Quick reduction of mean bias over all initializations.
+
+    Args:
+        hind (xr.object): hindcast.
+        bias (xr.object): bias.
+        dim (str): Time dimension name in bias.
+
+    Returns:
+        xr.object: bias reduced hind
+
+    """
+    bias_reduced_hind = hind - bias.mean(dim)
+    return bias_reduced_hind
+
+
+def _mean_bias_reduction_properly(hind, bias, dim):
+    """Reduce mean bias from all but the given initialization (cross-validation).
+
```

Comment:
Maybe add a note here:

```
.. note::

    This method follows Jolliffe 2011. For a given initialization, bias is computed over all other initializations, excluding the one in question. This calculated bias is removed from the given initialization, and then the process proceeds to the following one.
```

Just to make it simple on anybody looking at this in the future.